client/app/components/category_list.vue

```diff
@@ -0,0 +1,5 @@
+<template lang="pug">
+    main(role="main")
```

Comment:
The `role=main` attribute is useless here as the `main` tag have this implicit semantic.